presto-main/src/main/java/com/facebook/presto/sql/planner/InputExtractor.java

```diff
@@ -107,6 +108,14 @@ public Void visitSpatialJoin(SpatialJoinNode node, Context context)
             return null;
         }
 
+        @Override
+        public Void visitAggregation(AggregationNode node, Context context)
+        {
+            context.setExtractStatistics(true);
```

Comment:
I'm just curious - what is the rationale behind not extracting stats for some plan nodes? Too much noise?

Reply:
Yes, IMO that was the reason. I mostly agree - join & aggs are primary use cases where table stats would be very helpful for debugging behavior